.gitignore

```diff
@@ -1,2 +1,135 @@
 docs/book
-.env*
+
+# Logs
```

Comment:
Should this also remove the other .gitignore? Although nested .gitignores work, but that would imply that this shouldn't be changed.